src/server/db/models/github-user.js

```diff
@@ -9,7 +9,7 @@ import db from '../index';
  *   updated_at: update date
  *   last_login_at: last login date
  */
-export const GitHubUser = db.Model.extend({
+db.model('GitHubUser', {
```

Comment:
Is it just the ownership of the name that you find problematic? Why not make the model module export `name` and a `schema` properties?

**src/server/db/index.js**
```
import * as models from './models';
...
Object.keys(models).forEach((key) => {
  db.model(models[key]['name'], models[key]['schema']);
});
```

**src/server/db/models/index.js**
```
import * as GitHubUser from './github-user';

export {
  GitHubUser as GitHubUser
};

```

**src/server/db/models/github-user.js**
```
export const name = "GitHubUser";

export const schema = {
  tableName: 'github_user',
  hasTimestamps: true
};
```